installer/delta-engine/delta-engine.yaml

```diff
@@ -0,0 +1,189 @@
+        Bucket: !Ref S3BUCKETNAME
+        Key: !Sub ${RELEASEVERSION}/${Environment}/deltaengine/asset-sender-1.0-SNAPSHOT.jar
+      Description: Lambda function for asset shipping and delta engine.
+      MemorySize: 512
+      Timeout: 900
+      Handler: com.paladincloud.assetsender.AssetDoneHandler
+      Runtime: java21
+      Architectures:
+        - x86_64
+      EphemeralStorage:
+        Size: 512
+      Environment:
+        Variables:
+          ASSET_SHIPPER_DONE_SQS_URL: !Ref SQSAssetShippingDone
+          JAVA_TOOL_OPTIONS: '--enable-preview'
+          SUBSYSTEM_NAME: ASSET_SHIPPER
+          ENABLED_DELTA_ENGINE_PLUGINS: gcp
+          ASSET_MAPPING_DONE_QUEUE_URL: !Ref SQSAssetMappingDone
+      EventInvokeConfig:
+        MaximumEventAgeInSeconds: 21600
+        MaximumRetryAttempts: 2
+      PackageType: Zip
+      Role: !GetAtt DeltaEngineLambdaExecutionRole.Arn # Attach the role to the Lambda function
+      VpcConfig:
+        SecurityGroupIds:
+          - !Ref SecurityGroup
+        SubnetIds:
+          - !Ref Subnet1
+          - !Ref Subnet2
+        Ipv6AllowedForDualStack: false
+      Events:
+        SQS1:
+          Type: SQS
+          Properties:
+            Queue: !GetAtt SQSAssetMappingDone.Arn
+            BatchSize: 10
+      RuntimeManagementConfig:
+        UpdateRuntimeOn: Auto
+
+  LambdaLogGroup:
+    Type: AWS::Logs::LogGroup
+    Properties: 
+      LogGroupName: !Sub "/aws/lambda/svc-${Environment}-asset-shipper"
+      RetentionInDays: 14
+
+  SQSAssetShippingDone:
+    Type: AWS::SQS::Queue
+    Properties:
+      QueueName: !Sub "svc-${Environment}-asset-shipping-done.fifo"  # Use !Sub for environment substitution
+      FifoQueue: true
+      ContentBasedDeduplication: true      
+      SqsManagedSseEnabled: true
+      VisibilityTimeout: 900
+
+  SQSAssetMappingDone:
+    Type: AWS::SQS::Queue
+    Properties:
+      QueueName: !Sub "svc-${Environment}-asset-mapping-done.fifo"  # Use !Sub for environment substitution
+      FifoQueue: true
+      SqsManagedSseEnabled: true
+      ContentBasedDeduplication: true
+      VisibilityTimeout: 900
+
+  SecurityGroup:
+    Type: AWS::EC2::SecurityGroup
+    Properties:
+      GroupDescription: Security group for Lambda VPC access
+      VpcId: !Ref VpcId
+      SecurityGroupIngress:
+        - IpProtocol: tcp
+          FromPort: 80
+          ToPort: 80
+          CidrIp: !Ref CidrIp
+        - IpProtocol: tcp
+          FromPort: 443
+          ToPort: 443
+          CidrIp: !Ref CidrIp
+        - IpProtocol: tcp
+          FromPort: 3306
+          ToPort: 3306
+          CidrIp: !Ref CidrIp
+
+Outputs:
+  LambdaFunctionArn:
+    Description: The ARN of the Lambda function.
+    Value: !GetAtt svcassetshipper.Arn
+    Export:
+      Name: LambdaFunctionArn
+
+  AssetShipperSQSQueueURL:
+    Description: The URL of the svc-asset-shipping-done.fifo SQS queue.
+    Value: !Ref SQSAssetShippingDone
+    Export:
+      Name: AssetShipperSQSQueueURL
```

Comment:
need to standardize all variable to reflect the service name, or queue name. 

e.g., AssetDeltaEngineDoneSqs or similar, just needs to follow the same convension.